src/Logic.ml

```diff
@@ -29,7 +29,18 @@ let all_supported =
              (BooleanComponents.all @ IntegerComponents.peano) ;
            |] ;
            sample_set_size_multiplier = 8
-         }]
+         } ; {
+           name = "ALIA" ;
+           components_per_level = [|
```

Comment:
Could you leave a `FIXME` here:

`(* FIXME: Determine levels of ArrayComponents for hybrid enumeration *)`